internal/policy/verify.go

```diff
@@ -184,20 +184,34 @@ func VerifyRelativeForRef(ctx context.Context, repo *git.Repository, initialPoli
 	// Verify each entry, looking for a fix when an invalid entry is encountered
 	var invalidEntry *rsl.ReferenceEntry
 	var verificationErr error
+
+	targetRefEntiresCount := 0
+
+	for _, entry := range entries {
+		if entry.RefName == target {
+			targetRefEntiresCount++
+		}
+	}
+
+	if strings.HasPrefix(target, gitinterface.TagRefPrefix) && targetRefEntiresCount > 1 {
+		return ErrMultipleTagRSLEntriesFound
+	}
+
 	for len(entries) != 0 {
 		if invalidEntry == nil {
 			// Pop entry from queue
 			entry := entries[0]
 			entries = entries[1:]
 
 			slog.Debug(fmt.Sprintf("Verifying entry '%s'...", entry.ID.String()))
-
-			slog.Debug("Checking if entry is for policy staging reference...")
-			if entry.RefName == PolicyStagingRef {
-				continue
-			}
-			slog.Debug("Checking if entry is for policy reference...")
-			if entry.RefName == PolicyRef {
+			slog.Debug("Checking reference of entry...")
+
+			switch entry.RefName {
+			case PolicyStagingRef:
+				// If we encounter a policy-staging entry, we alert the user, but ignore it for gittuf verification
+				slog.Debug("Entry is for the policy staging reference...")
```

Comment:
nit: we use the `...` trailer when something is about to happen, such as "Checking X...", but to indicate a result, we don't have punctuation at the end (unless it's something like "Verification successful!").

Reply:
Fixed